packages/manager/apps/container/src/container/nav-reshuffle/sidebar/navigation-tree/root.ts

```diff
@@ -17,141 +18,11 @@ const root: Node = {
   children: [
     bareMetalCloud,
     hostedPrivateCloud,
-    publicCloud,
-    webCloud,
     network,
+    publicCloud,
+    sunrise,
     telecom,
-    {
-      id: 'sunrise',
-      translation: 'sidebar_sunrise',
-      routing: {
-        application: 'sunrise',
-      },
-      count: false,
-      features: ['sunrise'],
-      separator: true,
-    },
-    {
```

Comment:
Not sure where it went but I think this is still needed (maybe in its own file)
For example `sidebar-link-billing` will be used by the guided tour